docs/how-to-guides/gitops-using-azure-arc-onboarding-component.md

```diff
@@ -0,0 +1,238 @@
+
+```console
+$ kubectl -n rook get pods -n azure-arc-onboarding
+NAME                       READY   STATUS      RESTARTS   AGE
+azure-arc-register-4s54j   0/1     Completed   0          81m
+```
+
+Azure Arc installs various Helm charts in the `azure-arc` namespace, verify that all the deployments are in
+`Running` state:
+
+```console
+$ kubectl -n azure-arc get pods
+NAME                                         READY   STATUS    RESTARTS   AGE
+cluster-metadata-operator-7cff574c4f-ld8cv   2/2     Running   0          83m
+clusterconnect-agent-6dfd867c68-629xg        3/3     Running   0          83m
+clusteridentityoperator-fd498bf96-dqlhr      2/2     Running   0          83m
+config-agent-bd647558d-f46xn                 2/2     Running   0          83m
+controller-manager-8676dcdc6-mv68p           2/2     Running   0          83m
+extension-manager-bcfd5b597-lsk85            2/2     Running   0          83m
+flux-logs-agent-6596f58c56-55l7r             1/1     Running   0          83m
+kube-aad-proxy-97bf4bcf8-g9djl               2/2     Running   0          83m
+metrics-agent-5b9b94754f-qf5q2               2/2     Running   0          83m
+resource-sync-agent-f8c7c6b6b-zvhx4          2/2     Running   0          83m
+```
+
+### Step 3: Create a Kubernetes configuration
+
+#### Create a configuration
+
+Once the Lokomotive cluster is onboarded on Azure Arc, we can proceed ahead with creating a configuration for
+deploying the GitOps agent.
+
+```bash
+az k8s-configuration create \
+  --name cluster-config \
+  --cluster-name "${CLUSTER_NAME}" \
+  --resource-group "${RG_NAME}" \
+  --operator-instance-name cluster-config \
+  --operator-namespace cluster-config \
+  --repository-url https://github.com/ipochi/azure-arc-nginx-demo \
+  --scope cluster \
+  --cluster-type connectedClusters \
+  --operator-params='--git-branch=main'
+```
+
+- `name` - name of the Kubernetes configuration you want.
+- `cluster-name` - name of the cluster provided in `azure-arc-onboarding` component configuration.
+- `resource-group` - Azure Resource group name provided in the `azure-arc-onboarding` component configuration.
+- `operator-instance-name` - name of the Operation instance.
+- `repository-url` - name of the Git repository.
+- `scope` - scope of the operator. Accepted values: `cluster` or `namespace`.
+- `cluster-type` - Arc cluster type. Accepted values: `connectedClusters` or `managedClusters`.
+   **NOTE**: If the Lokomotive cluster is deployed on AKS, then the value of `cluster-type` is `managedClusters`
+- `operator-params` - Additional parameters to pass to the Operator. By default, the GitOps agent tracks and
+   syncs with the `master` branch. In this case, changing default branch to `main`.
+
+This creates two Deployments in the provided namespace `cluster-config`, corresponding to the GitOps agent
+that Azure Arc uses:
+
+```bash
+kubectl get deployments -n cluster-config
```

Comment:
Do we want to show this as a console output?

(also applies to other examples in this document)
```suggestion
```console
$ kubectl get deployments -n cluster-config
```